cogs5e/models/homebrew/bestiary.py

```diff
@@ -382,8 +383,9 @@ def parse_critterdb_traits(data, key):
                             raise ValueError('Invalid YAML')
                         elif 'name' not in atk:
                             atk['name'] = name
+                        Attack.from_dict(deepcopy(atk))
```

Comment:
Why is the deepcopy necessary here?

Reply:
Removed in later version. Was necessary at the time because Attack.from_dict() was being run twice and has destructive popping.